server/database/db_connection.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require('mongoose');
+
+// load .env only on development
+if (!process.env.TRAVIS && !process.env.ENV !== 'production') {
+  require('dotenv').config()
+}
+
+const dbConnection = () => {
+  let { MONGOURI: mongoURI } = process.env;
+
+  if (process.env.NODE_ENV === 'test') {
+    // change MONGOURI to testDBurl
+    mongoURI = process.env.MONGOURI_TEST; 
+  }
+  // create DB connection
+  return mongoose.connect(
```

Comment:
I think we should use the ```then``` and ```catch``` blocks to let us know if connection is successful or not.
At the very least, we should catch the error if it happens, perhaps restart the process.

For now just add a catch block with a message and the error stack.